src/CompletionCommand.php

```diff
@@ -15,12 +15,15 @@ class CompletionCommand extends SymfonyCommand
      */
     protected $handler;
 
+    /**
+     *  Bash completion command description.
+     */
+    protected static $defaultDescription = 'BASH completion hook.';
+
     protected function configure()
     {
         $this
             ->setName('_completion')
-            ->setDefinition($this->createDefinition())
```

Comment:
Accidental removal? This breaks the command :)

Reply:
added back.